Modules/IO/PNG/src/itkPNGImageIO.cxx

```diff
@@ -266,7 +266,7 @@ void PNGImageIO::Read(void *buffer)
 
   auto rowbytes = static_cast<SizeValueType>( png_get_rowbytes(png_ptr, info_ptr) );
   auto * tempImage = static_cast< unsigned char * >( buffer );
-  auto * row_pointers = new png_bytep[height];
+  std::unique_ptr<png_bytep[]> row_pointers (new png_bytep [height]);
```

Comment:
Could probably even be `const std::unique_ptr<png_bytep[]>` (to express that you won't reset the pointer afterwards), but that's no show-stopper for me